src/bicep/examples/newWorkload/README.md

```diff
@@ -16,19 +16,19 @@ The docs on Azure virtual networking:  <https://docs.microsoft.com/en-us/azure/v
 ## Pre-requisites
```

Comment:
I made updates to the README. Let me know if these are changes are correct by resolving this conversation.